web/src/beta/components/fields/CameraField/index.tsx

```diff
@@ -0,0 +1,313 @@
+import { useCallback, useState } from "react";
+
+import Button from "@reearth/beta/components/Button";
+import NumberInput from "@reearth/beta/components/fields/common/NumberInput";
+import Icon from "@reearth/beta/components/Icon";
+import * as Popover from "@reearth/beta/components/Popover";
+import Slider from "@reearth/beta/components/Slider";
+import Text from "@reearth/beta/components/Text";
+import { useT } from "@reearth/services/i18n";
+import { styled, useTheme } from "@reearth/services/theme";
+
+import Property from "..";
+
+type CameraInput = {
+  name: string;
+  field: keyof CameraValue;
+};
+// Constants
```

Comment:
Since there are more than one constant defined, can you move them into a constants.ts file? 🙏🏻